python/drned_xmnr/op/filtering.py

```diff
@@ -356,184 +355,312 @@ def event_generator(consumer):
         consumer.close()
 
 
```

Comment:
I'm not sure 'LS' instead of 'LogState' and 'S' instead of 'State' makes it easier to follow the code. Are you afraid lines get too long?

Reply:
I named the states ...S/...LS to make them differ from the old states when I was rewriting the code; but now it does not really make sense, I'll change S to State.